extensions/wikia/CuratedContent/CuratedContentValidator.class.php

```diff
@@ -63,7 +70,20 @@ private function validateImage( $sectionOrItem ) {
 		}
 	}
 
-	private function validateSection( $section ) {
+	public function validateSectionItems( $section ) {
+		if ( !empty( $section['items'] ) && is_array( $section['items'] ) ) {
+			foreach ($section['items'] as $item) {
+				$this->validateSectionItem( $item );
+			}
+		} else {
+			// if section doesn't have any items and it's not Featured Section, it's an error
+			if ( empty( $section['featured'] ) ) {
```

Comment:
Why do we need this check if there is a separate method for featured section, `validateFeaturedSectionItems`?